File: year2020/month08/day0831/ms_0831.cpp
#include <iostream>

using namespace std;

int main() {
	char data[3][5];
	
	for (int i = 0; i < 3; i++) {
		for (int j = 0; j < 5; j++) {
			char temp;

			scanf_s("%c", &temp);
			
			if ('A' <= temp && temp <= 'Z')
				data[i][j] = temp;
			else j--;
		}
	}

	for (int i = 0; i < 3; i++) {
		for (int j = 0; j < 5; j++) {
			cout << char(data[i][j] + 32);
			
			if (j != 5)
				cout << ' ';
		}

		cout << endl;
	}

	return 0;
}

File: year2020/month08/day0831/ms_0831.cs
using System;

namespace algorithm_46
{
    class Program
    {
        static void Main(string[] args)
        {
            char[,] data = new char[3, 5];

            for(int i = 0; i < 3; i++) 
            {
                string[] inputs = Console.ReadLine().Split(' ');
                int j = 0;

                foreach(var word in inputs) 
                {
                    char temp = Convert.ToChar(word);

                    if('A' <= temp && temp <= 'Z') 
                    {
                        data[i, j] = temp;
                        j++;
                    } 
                    else 
                    {
                        i--;
                        continue;
                    }
                }
            }

            for(int i = 0; i < 3; i++) 
            {
                for(int j = 0; j < 5; j++) 
                {
                    Console.Write($"{Convert.ToChar(data[i, j] + 32)}");

                    if(j != 5)
                        Console.Write(' ');
                }

                Console.WriteLine();
            }
        }
    }
}
